packages/data-provider/test/utils/dictionarySchemasTestData.ts

```diff
@@ -0,0 +1,213 @@
+import { Dictionary } from '@overture-stack/lyric-data-model';
+import { SchemaDefinition, ValueType } from '@overturebio-stack/lectern-client/lib/schema-entities.js';
+
+const sampleDictionaryData: SchemaDefinition[] = [
```

Comment:
You may want to create a dir of `test/fixtures` that includes test dictionaries

Reply:
This dictionary could be called `dictionarySportStats`

Reply:
Moved this file to `fixtures/` folder and changed it's variable names